### PR TITLE
Removed MS.Owin.Testing and added OwinHttpMessageHandler

### PR DESCRIPTION
**Pull request by jchannon**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/NancyFx/Nancy/pulls) open
- [x] I have verified that I am following the Nancy [code style guidelines](https://github.com/NancyFx/Nancy/blob/45238076ad0b7f6ecabd6bae8469e30458d02efe/CONTRIBUTING.md#style-guidelines)
- [x] I have provided test coverage for my change (where applicable)

### Description
Removed MS Owin Testing and uses @damianh OwinHttpMessageHandler

### CODE REVIEW

**Comment by thecodejunkie** [comment]
:x: https://ci.appveyor.com/project/khellang/nancy/build/2.0.0-ci000179-yhmkfgfe#L272

**Comment by jchannon** [comment]
go go go

File: test/Nancy.Owin.Tests/AppBuilderExtensionsFixture.cs [hunk 1]
```diff
@@ -58,23 +61,35 @@ public void When_host_Nancy_via_IAppBuilder_should_read_X509Certificate2()
...
 
-                var result = server.Invoke(env);
-                result.Wait();
+            app.UseNancy(opts =>
+            {
+                opts.Bootstrapper = bootstrapper;
+                opts.EnableClientCertificates = true;
+            });
+
+
```
Comment (thecodejunkie) [inline test/Nancy.Owin.Tests/AppBuilderExtensionsFixture.cs]: :scream:

**Comment by thecodejunkie** [comment]
Tiny change then it will be ready to be pulled

**Comment by jchannon** [comment]
done

**Pull request merged**